django_tables2/utils.py

```diff
@@ -360,9 +360,14 @@ def get_field(self, model):
                 field = model._meta.get_field(bit)
             except FieldDoesNotExist:
                 break
-            if hasattr(field, 'rel') and hasattr(field.rel, 'to'):
-                model = field.rel.to
-                continue
+
+            if hasattr(field, 'remote_field'):
+                rel = getattr(field, 'remote_field', None)
+                model = getattr(rel, 'model', model)
+            else:
+                # Since django 1.9 `rel` attribute was replaced by `remote_field`
```

Comment:
Can you change this comment in such a way that when we remove support for the last version this shim is intended for we only have to look for the version number we are removing support for?